spec/ruby/core/io/read_spec.rb

```diff
@@ -104,6 +104,16 @@
     str = IO.read(@fname, encoding: Encoding::ISO_8859_1)
     str.encoding.should == Encoding::ISO_8859_1
   end
+
+  platform_is :windows do
+    it "reads the file in text mode" do
+      # 0x1A is CTRL+Z and is EOF in Windows text mode.
+      content = "\x1Abbb"
+      rm_r(@fname)
+      File.binwrite(@fname, content)
+      IO.read(@fname).bytesize.should < content.bytesize
```

Comment:
Why not simply assert the result to be empty?
```suggestion
      File.binwrite(@fname, "\x1Abbb")
      IO.read(@fname).should.empty?
```

Reply:
Ah that's much better. I over complicated it 😅